src/System.Threading.Tasks/pkg/aot/System.Threading.Tasks.pkgproj

```diff
@@ -7,5 +7,11 @@
     <PreventImplementationReference>true</PreventImplementationReference>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
The placeholder is used to tell nuget that your package does support a given framework, but the asset is located in a different assembly.